app/src/main/kotlin/co/netguru/baby/monitor/client/feature/babycrynotification/CryingActionIntentService.kt

```diff
@@ -45,16 +45,16 @@ class CryingActionIntentService : IntentService(NAME) {
             .setComponentName(ClientHomeActivity::class.java)
             .setGraph(R.navigation.client_home_nav_graph)
             .setDestination(R.id.clientLiveCamera)
+            .setArguments(showSnoozeDialogArg())
             .createTaskStackBuilder()
             .startActivities()
     }
 
-    private fun handleSnoozeAction() {
-        dataRepository.updateChildSnoozeTimestamp(
-            System.currentTimeMillis() + FIVE_MINUTES_SNOOZE_TIME
-        )
-            .subscribe { Timber.i("Notification snooze timestamp updated") }
-            .addTo(disposables)
+    private fun showSnoozeDialogArg(): Bundle {
+        return Bundle()
+            .apply {
+                putBoolean(SHOULD_SHOW_SNOOZE_DIALOG, true)
+            }
```

Comment:
`private fun showSnoozeDialogArg() =bundleOf(SHOULD_SHOW_SNOOZE_DIALOG to true)`